.github/workflows/catch.yml

```diff
@@ -0,0 +1,33 @@
+name: catch
```

Comment:
ugh I hate yaml, I spent a day last week trying to write one line, and eventually got someone else to write it for me

Reply:
I needed that person today 😡